projects/plugins/jetpack/class.jetpack-connection-banner.php

```diff
@@ -251,7 +239,7 @@ public static function enqueue_connect_button_scripts() {
 				'apiSiteDataNonce'      => wp_create_nonce( 'wp_rest' ),
 				'buttonTextRegistering' => __( 'Loading...', 'jetpack' ),
 				'jetpackApiDomain'      => $jetpack_api_url['scheme'] . '://' . $jetpack_api_url['host'],
-				'forceVariation'        => $force_variation,
+				'forceVariation'        => 'original',
```

Comment:
Do we still need this line? It seems that it's not used anymore

Reply:
we don't! thanks